src/pages/_proxied-dot-io/consul/use-cases/[slug].tsx

```diff
@@ -0,0 +1,242 @@
+            text: 'Explore all',
+          }}
+          cardsPerRow={3}
+          cards={tutorialCards.map((card) => {
+            return {
+              eyebrow: card.eyebrow,
+              link: {
+                url: card.link,
+                type: 'inbound',
+              },
+              heading: card.heading,
+              description: card.description,
+              products: card.products,
+            }
+          })}
+        />
+
+        <IoCardContainer
+          label="Docs"
+          cta={{
+            url: documentationLink ? documentationLink : '/docs',
+            text: 'Explore all',
+          }}
+          cardsPerRow={3}
+          cards={documentationCards.map((card) => {
+            return {
+              eyebrow: card.eyebrow,
+              link: {
+                url: card.link,
+                type: 'inbound',
+              },
+              heading: card.heading,
+              description: card.description,
+              products: card.products,
+            }
+          })}
+        />
+      </div>
+
+      <div className={s.callToAction}>
+        <IoUsecaseCallToAction
+          theme="dark"
+          brand="consul"
+          heading={callToActionHeading}
+          description={callToActionDescription}
+          links={callToActionLinks.map((link) => {
+            return {
+              text: link.title,
+              url: link.link,
+            }
+          })}
+          pattern="/consul/img/usecase-callout-pattern.svg"
+        />
+      </div>
+
+      {_videoCallout ? (
+        <div className={s.videoCallout}>
+          <IoVideoCallout
+            youtubeId={_videoCallout.youtubeId}
+            thumbnail={_videoCallout.thumbnail.url}
+            heading={_videoCallout.heading}
+            description={_videoCallout.description}
+            person={{
+              avatar: _videoCallout.personAvatar?.url,
+              name: _videoCallout.personName,
+              description: _videoCallout.personDescription,
+            }}
+          />
+        </div>
+      ) : null}
+    </>
+  )
+}
+
+export async function getStaticPaths() {
+  const query = proxiedRivetClient('consul')
+  const { allConsulUseCases } = await query({
+    query: useCasesQuery,
+  })
+
+  return {
+    paths: allConsulUseCases.map((page) => {
+      return {
+        params: {
+          slug: page.slug,
+        },
+      }
+    }),
+    fallback: 'blocking',
+  }
+}
+
+export async function getStaticProps({ params }) {
+  const { slug } = params
+
+  const query = proxiedRivetClient('consul')
+  const { allConsulUseCases } = await query({
+    query: useCasesQuery,
+  })
+
+  const page = allConsulUseCases.find((page) => page.slug === slug)
+
+  if (!page) {
+    return { notFound: true }
+  }
+
+  return {
+    props: {
+      data: page,
+    },
+    revalidate:
+      process.env.HASHI_ENV === 'production'
+        ? process.env.GLOBAL_REVALIDATE
+        : 10,
```

Comment:
Do we have `GLOBAL_REVALIDATE` defined anywhere?  💭

Reply:
Ah sorry, I forgot Vault and Consul used this!